packages/features/ee/workflows/lib/variableTranslations.ts

```diff
@@ -21,9 +21,12 @@ export function getTranslatedText(text: string, language: { locale: string; t: T
     variables?.forEach((variable) => {
       const regex = new RegExp(variable, "g"); // .replaceAll is not available here for some reason
       const translatedVariable = originalVariables.includes(variable.toLowerCase().concat("_workflow"))
-        ? language.t(variable.toLowerCase().concat("_workflow")).replace(/ /g, "_").toLocaleUpperCase()
```

Comment:
This should work just as replaceAll doesn't it?

Reply:
Good point, I made some changes and I am using now / /g instead of replaceAll